### PR TITLE
flyingcircus:3.2.0

### PR DESCRIPTION
**Pull request by Tetragramm**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: Updated the package to work with typst 0.12.0, as well as updating dependencies.
In addition added new features to bring it up to par with the LaTeX template, and a bit beyond.
Playbook functions, and new short NPC and Airship profiles.

I have read and followed the submission guidelines and, in particular, I
- [X] selected a name that isn't the most obvious or canonical name for what the package does
- [X] added a `typst.toml` file with all required keys
- [X] added a `README.md` with documentation for my package
- [X] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [X] tested my package locally on my system and it worked
- [X] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/flyingcircus/3.2.0/README.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+#FCPlane(read("My Plane_stats.json"), Nickname:"My First Plane")
+```
+
+## Usage
+
+The first thing is the FlyingCircus style.
+
+```typ
+#import "@preview/flyingcircus:3.0.0": *
```
Comment (elegaanz) [inline packages/preview/flyingcircus/3.2.0/README.md:40]:
```suggestion
#import "@preview/flyingcircus:3.2.0": *
```

**Review by elegaanz — COMMENTED** [review]
For info [tablex is not yet compatible with Typst 0.12](https://github.com/PgBiel/typst-tablex/issues/135) and your package still reports warning because of that. If you are fine with it and want to update tablex later, I can merge this PR already (once the one outdated import is fixed). If it bothers you, you can leave the PR open and push a new commit to update tablex once a new version is available.

**Comment by Tetragramm** [comment]
> For info [tablex is not yet compatible with Typst 0.12](https://github.com/PgBiel/typst-tablex/issues/135) and your package still reports warning because of that. If you are fine with it and want to update tablex later, I can merge this PR already (once the one outdated import is fixed). If it bothers you, you can leave the PR open and push a new commit to update tablex once a new version is available.

This inspired me to double-check, and the problems I was using tablex to work around have either been fixed, or I've figured out how to do them correctly.
So I have removed the tablex dependency entirely.

**Comment by elegaanz** [comment]
Nice :) Thank you!

**Pull request merged**